README.md

```diff
@@ -1,20 +1,34 @@
 # Code + Learn
 
-A series of workshop sprints for Node.js.
+Code + Learn is an in-person (and free) event series where developers receive mentoring from Node.js Core contributors to learn the step-by-step process of contributing to Node.js Core.
```

Comment:
I think right now this file is wrapping lines at 80 characters, it might be good to keep that consistent